src/main/java/com/google/jenkins/plugins/storage/AbstractUpload.java

```diff
@@ -330,8 +330,8 @@ public String getPathPrefix() {
   }
 
   /**
-   * Gives all registered {@link AbstractUpload}s.
-   * See: https://wiki.jenkins-ci.org/display/JENKINS/Defining+a+new+extension+point
+   * Gives all registered {@link AbstractUpload}s. See:
```

Comment:
I missed this in my review of #73 because I didn't check to see that any of the files were reformatted